cfgov/ask_cfpb/jinja2/ask-cfpb/landing-page.html

```diff
@@ -49,50 +49,789 @@ <h2 class="u-show-on-mobile u-mt45">
 
             <div class="u-hide-on-mobile question-categories u-mb15">
 
+                
                 {#
-                    TODO: This can use the standard card-group.html template
-                          if the partially styled links are removed.
+                    TODO:   This can use the standard card-group.html template
+                            if the partially styled links are removed.
                 #}
+
                 <div class="o-card-group o-card-group__column-2">
                     <div class="o-card-group_cards">
 
-                    {% for portal in portal_cards %}
+                        {% if page.language == 'en'%}
+                        <article class="m-card">
+                            <h2 class="m-card_heading">
+                                <a class="" href="/consumer-tools/auto-loans/">
+                                    {{ svg_icon( "car" ) }}
+                                    <span>Auto loans</span>
+                                </a>
+                            </h2>
+
+                            <ul class="m-list">
+                                
+                                    <li class="m-list_item">
+                                    <a class="a_link a-link__partially-styled" data-pretty-href="cfpb.gov/askcfpb/763" href="/ask-cfpb/what-effect-will-shopping-for-an-auto-loan-have-on-my-credit-en-763/">
```

Comment:
Per my other comment just now about `data-pretty-href`, these should be attended automatically by the Django middleware, so it would be better not to add them manually in the template.